elfi/methods/mcmc.py

```diff
@@ -415,4 +417,4 @@ def metropolis(n_samples, params0, target, sigma_proposals, seed=0):
 
     logger.info(
         "{}: Total acceptance ratio: {:.3f}".format(__name__, float(n_accepted) / n_samples))
-    return samples[1:, :]
+    return samples[(1+warmup):, :]
```

Comment:
No need for +1 here, since one would typically not count the 0 as 1 in `warmup`, and so giving `warmup=500` would already point to `samples[500:, :]` here.

Reply:
Sorry, I forgot that the +1 was accounting for the initial guess.